src/pages/Home/App.css

```diff
@@ -0,0 +1,379 @@
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+/* @import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;600;700&family=DM+Sans:opsz,wght@9..40,300;9..40,400;9..40,500;9..40,600;9..40,700;9..40,800&family=Gochi+Hand&display=swap'); */
+
+
+@font-face {
+    font-family: vag;
+    src: url("./fonts/VAG\ Rounded\ Regular.ttf");
+}
+
+/* BackGround */
+.app{
+    background: linear-gradient(180deg, rgba(255, 255, 255, 0.50) 0%, rgba(251, 241, 235, 0.96) 87.19%, #FFF 94.34%);
+ 
+    background-blend-mode: screen, normal;
+    
+    overflow: hidden;
+    background-color: #fff;
+}
+/* background: url(<path-to-image>), lightgray 0% 0% / 50px 50px repeat, linear-gradient(180deg, rgba(255, 255, 255, 0.50) 0%, rgba(251, 241, 235, 0.96) 87.19%, #FFF 94.34%); */
+
+.DM_Sans{
+    font-family: "DM Sans";
+}
+.vag{
+    font-family: vag;
+    /* font-weight: 300; */
+}
+.Quicksand{
+    font-family: "Quicksand";
+}
+
+.Gochi_Hand{
+    font-family: "Gochi Hand";
+}
+
+.shadow {
+    box-shadow: 0px 12px 36px 0px rgba(0, 0, 0, 0.06);
+}
+
+.shadow_1 {
+    box-shadow: 0px 12px 6px 0px rgba(80, 79, 79, 0.06);
+}
+
+/*  Hero.jsx  */
+.bg_blur {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 60%;
+    z-index: -1;
+    /* border-radius: 50%; */
+    background: #fdfdfd;
+    /* background-color: red; */
+    /* background: #ffff; */
+    filter: blur(400px);
+}
+
+
+.extra_width {
+    width: calc(100% + 70px)
+}
+
+.desc > * {
+    font-weight: 400px !important;
+    font-size: 16px;
+    display: inline;
+    line-height: 22px;
+    width: fit-content;
+}
+
+/*  Brands.jsx  */
+
+@keyframes slider {
+    from{
+        transform: translateX(0);
+    }
+    to{
+        transform: translateX(-100%);  
+    }
+}
+
+
+.brands {
```

Comment:
this class is unused, and can be removed - kindly look for other instances of unused classes

Reply:
This class is used inside brands section

Reply:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/21141664/904fe06a-ebb2-442b-86cd-570f7e0c404f)
****